package.json

```diff
@@ -50,7 +50,8 @@
   },
   "eslintIgnore": [
     "**/templates/plugin.js",
-    "playground/app.vue"
+    "playground/app.vue",
+    "lib/cypress/**/*"
```

Comment:
@alvarosabu this is the solution for the famous eslint issue?

Reply:
Actually no this didn't work, I had to do it with an inline disable